README.md

```diff
@@ -1,5 +1,5 @@
-1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task_generate_table_DOM/)
+1. Replace https://github.com/sinner1993/js_task_generate_table_DOM with your Github username in the link
+    - [DEMO LINK](https://sinner1993.github.io/js_task_generate_table_DOM/)
```

Comment:
The link in PR description is wrong now, add this demo link instead 
<img width="600" alt="image" src="https://github.com/user-attachments/assets/e3b7ddde-f4e6-4645-b71a-317d609d38ba">